<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QWidget" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>鱼腥味的游戏优化工具箱 V0.0.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/app.ico</normaloff>:/icons/app.ico</iconset>
  </property>
  <widget class="QWidget" name="widget_functions" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>720</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <widget class="QLabel" name="label_BackgroundActivityLimit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>限制后台活动</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_SystemServiceOptimization">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>系统服务优化</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_GameOptimizePowerPlan">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>游戏电源优化</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_PowerPlanLock">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>锁定电源计划（暂不可用）</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_AutoLimitAntiCheat">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>自动限制反作弊</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_AutoStartup">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>开机自启动</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_SystemSchedulerOptimization">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>系统调度优化</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="SwitchButton" name="switchButton_SetSystemServiceOptimization"/>
     </item>
     <item row="6" column="1">
      <widget class="SwitchButton" name="switchButton_SetPowerPlanLock">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SwitchButton" name="switchButton_SetAutoStartup"/>
     </item>
     <item row="7" column="1">
      <widget class="SwitchButton" name="switchButton_SetLimitBackgroundActivity"/>
     </item>
     <item row="4" column="1">
      <widget class="SwitchButton" name="switchButton_SetAutoLimitAntiCheat"/>
     </item>
     <item row="9" column="1">
      <widget class="SwitchButton" name="switchButton_SetSystemSchedulerOptimization"/>
     </item>
     <item row="5" column="1">
      <widget class="SwitchButton" name="switchButton_SetGameOptimizePowerPlan"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_NetWorkDelayOptimization">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>网络延迟优化</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="SwitchButton" name="switchButton_SetNetworkDelayOptimization"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget_GameProcess">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>451</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>24</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>游戏名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>优化状态</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>359</y>
      <width>311</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QLabel" name="label_note">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>309</width>
      <height>138</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/imgs/qr_code.png&quot; width=&quot;800&quot; height=&quot;300&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;本软件&lt;a href=&quot;https://www.baidu.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#004275;&quot;&gt;Github开源地址&lt;/span&gt;&lt;/a&gt;，喜欢的给个Star吧！！！&lt;/p&gt;&lt;p&gt;大学生为爱发电不易，&lt;a href=&quot;https://www.baidu.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#004275;&quot;&gt;请我喝瓶水吧~&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;有需要鞋子的兄弟可以添加微信：&lt;a href=&quot;https://www.baidu.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#004275;&quot;&gt;Mrmuscle12138&lt;/span&gt;&lt;/a&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QAbstractButton</extends>
   <header>ui/components/switchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
